### PR TITLE
Bug [838] - XPaths for all Configuration menus needs to be changed ac…

### PR DESCRIPTION
**Pull request by manishagupta592**
…cording to New UI.

### CODE REVIEW

File: src/main/java/agentcommissions/AgentCommissionProcessPage.java [hunk 1]
```diff
@@ -369,10 +369,7 @@ public void clickAgentCommissionProcess() throws IOException{
     	AgentCommissionProcessPage sp=new AgentCommissionProcessPage(driver);
         log.info("Click on Agent Commission Process Link in left menu bar");
         WaitClass.sleep(10000);
-        WaitClass.WaitForElementisDisplay(driver, 10, clickAgentCommissionProcess);
-        Assert.assertTrue(clickAgentCommissionProcess.isDisplayed());
-        clickAgentCommissionProcess.click();
-        
+        driver.navigate().to("http://localhost:8080/jbilling/config/partnerCommission");
```
Comment (vikasbo) [inline src/main/java/agentcommissions/AgentCommissionProcessPage.java]: @manishagupta592 Please remove portion `http://localhost:8080/jbilling', this part is fixed and it should come from configuration file.

File: src/main/java/agentcommissions/AgentConfigurationPluginPage.java [hunk 2]
```diff
@@ -162,10 +162,7 @@ public void clickPlugin() throws IOException{
     	AgentConfigurationPluginPage sp=new AgentConfigurationPluginPage(driver);
         log.info("Click on Plugin after successful login");
         WaitClass.sleep(10000);
-        WaitClass.WaitForElementisDisplay(driver, 10, clickPlugin);
-        Assert.assertTrue(clickPlugin.isDisplayed());
-        clickPlugin.click();
-        
+        driver.navigate().to("http://localhost:8080/jbilling/plugin/list");
```
Comment (vikasbo) [inline src/main/java/agentcommissions/AgentConfigurationPluginPage.java]: @manishagupta592 same as above.

File: src/main/java/agentcommissions/AgentCommissionProcessPage.java (hunk 1 above)
Comment (manishagupta592) [inline src/main/java/agentcommissions/AgentCommissionProcessPage.java]: @vikasbo Done.

File: src/main/java/agentcommissions/AgentConfigurationPluginPage.java (hunk 2 above)
Comment (manishagupta592) [inline src/main/java/agentcommissions/AgentConfigurationPluginPage.java]: @vikasbo Done.

**Pull request merged**